documentation/API_documentation/device-data-volume-API-Readiness-Checklist.md

```diff
@@ -0,0 +1,18 @@
+# API Readiness Checklist
+
+Checklist for device-data-volume v0.1.0-alpha.1 in r1.1.
+
+| Nr | API release assets  | alpha | release-candidate |  initial<br>public | stable<br> public | Status | Reference information |
+|----|----------------------------------------------|:-----:|:-----------------:|:-------:|:------:|:----:|:----:|
+|  1 | API definition                               |   M   |         M         |    M    |    M   |  Y   | [relative link](/code/API_definitions/device-data-volume.yaml) |
+|  2 | Design guidelines from Commonalities applied |   O   |         M         |    M    |    M   |  TBD |  |
+|  3 | Guidelines from ICM applied                  |   O   |         M         |    M    |    M   |  TBD |  |
```

Comment:
Link to the release which was used in this release could be added.

Reply:
I'll do that for the release candidate